charts/hivemq-edge/values.schema.json

```
@@ -32,9 +32,15 @@
       "enabled" : {
         "type" : "boolean"
       },
-      "volume" : {
-        "description": "Volume containing the license.",
-        "type" : "object"
+      "volume": {
+        "licenseSecretName" : {
+          "description": "Name of the secret where the license is stored.",
+          "type" : "string"
+        },
+        "licenseSecretKey" : {
+          "description": "Name of the key inside the secrete which contains the license.",
+          "type" : "string"
+        }
       },
```

Missing the type `object` for the `volume` value along with its properties, otherwise the schema validation won't work/complain. Should be like:

```suggestion
      "volume": {
        "properties" : {
          "licenseSecretName" : {
            "description": "Name of the secret where the license is stored.",
            "type" : "string"
          },
          "licenseSecretKey" : {
            "description": "Name of the key inside the secrete which contains the license.",
            "type" : "string"
          }
        },
        "type" : "object"
      },
```